go/adbc/driver/flightsql/utils.go

```diff
@@ -107,6 +115,9 @@ func adbcFromFlightStatusWithDetails(err error, header, trailer metadata.MD, con
 			}
 		default:
 			for _, value := range values {
+				if strings.HasPrefix(value, "Grpc-") {
+					continue
+				}
```

Comment:
why skip things with `Grpc-` prefix?

Reply:
I don't know why and how, but I was getting TextErrorDetails with key `trailer` and value `Grpc-Status` and `Grpc-Message` and `Grpc-Status-Details-Bin`. I don't think those values can be useful.